src/components/IGVBrowser.js

```diff
@@ -12,6 +12,14 @@ import { useCancellation, useOnMount, withDisplayName } from 'src/libs/react-uti
 import { knownBucketRequesterPaysStatuses, requesterPaysProjectStore } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
+
+const customReferences = [
+  {
+    name: 'MN908947.3',
+    referenceDefinition: { reference: { id: 'sarsCov2RefId.3', indexed: false, fastaURL: 'https://storage.googleapis.com/gcp-public-data--broad-references/sars-cov-2/MN908947.3/nCoV-2019.reference.fasta' } }
+  }
+]
```

Comment:
Since this is effectively a key/value map, an object may be a better representation, i.e.:
```
// Additional references supported by Terra that are not included in IGV
const customReferences = {
  'MN908947.3': {
    id: 'sarsCov2RefId.3', indexed: false,
    fastaURL: 'https://storage.googleapis.com/gcp-public-data--broad-references/sars-cov-2/MN908947.3/nCoV-2019.reference.fasta'
  }
}
```
This makes finding the reference by name a simple attribute reference instead of a `_.find`.